packages/suite-data/files/translations/zh.json

```diff
@@ -2,8 +2,8 @@
   "AMOUNT": "金额",
   "AMOUNT_EXCEEDS_MAX": "该金额超出了 {maxAmount} 的最大允许值。",
   "AMOUNT_IS_BELOW_DUST": "金额必须至少为 {dust}",
-  "AMOUNT_IS_LESS_THAN_RESERVE": "接收账户需要最低保留 {reserve} XRP 才能激活",
-  "AMOUNT_IS_MORE_THAN_RESERVE": "金额超过规定的不可动用储备金 ({reserve} XRP)。",
+  "AMOUNT_IS_LESS_THAN_RESERVE": "接收账户需要最低保留 {reserve} {symbol} 才能激活",
```

Comment:
you don't need to update translations files. They are autogenerated from https://crowdin.com/ where you may participate if you want.

Reply:
Hi @mroz22, thank you for the reminder. To simplify the demonstration process, I manually updated it. I have emailed Miroslav, and once Trezor agrees to add this feature to Suite, I will update this PR with tests, documentation, and internationalization (some parts are hardcoded in English).